src/pathfinding_service/service.py

```diff
@@ -264,8 +287,14 @@ def _validate_pfs_update(self, message: UpdatePFS) -> TokenNetwork:
 
     def on_pfs_update(self, message: UpdatePFS) -> None:
         token_network = self._validate_pfs_update(message)
+
         log.info("Received Capacity Update", **message.to_dict())
         self.database.upsert_capacity_update(message)
+
+        # Follow presence for the channel participants
+        self.matrix_listener.follow_address_presence(message.updating_participant, refresh=True)
+        self.matrix_listener.follow_address_presence(message.other_participant, refresh=True)
```

Comment:
Why do we have `refresh=True` here, but not in `_load_token_networks`?

Reply:
`_load_token_networks` is run before the matrix client is started, so just register interest in the address and this is taken care of when the client starts.

When we receive a new channel we have to register interest but also query the current state, which is what `refresh=True` does.